modAionImpl/src/org/aion/zero/impl/cli/Cli.java

```diff
@@ -225,6 +228,48 @@ public int call(final String[] args, final Cfg cfg) {
                         RecoveryUtils.printStateTrieSize(block_count);
                     }
                     break;
+
+                // Determines network config folder path
+                case "--network": {
+                    if (args.length == 2 && isValid(args[1])) {
+                        switch (args[1].toLowerCase()) {
+                            case "mainnet": {
+                                CfgAion.setNetwork("mainnet");
+                                CfgAion.setConfFilePath((BASE_PATH + "/config/mainnet/config.xml"));
+                                CfgAion.setGenesisFilePath((BASE_PATH + "/config/mainnet/genesis.json"));
+                                break;
+                            }
+                            case "conquest": {
+                                CfgAion.setNetwork("conquest");
+                                CfgAion.setConfFilePath((BASE_PATH + "/config/conquest/config.xml"));
+                                CfgAion.setGenesisFilePath((BASE_PATH + "/config/conquest/genesis.json"));
+                                break;
+                            }
+                            default: {
+                                CfgAion.setNetwork("mainnet");
+                                CfgAion.setConfFilePath((BASE_PATH + "/config/mainnet/config.xml"));
+                                CfgAion.setGenesisFilePath((BASE_PATH + "/config/mainnet/genesis.json"));
+                                break;
+                            }
```

Comment:
for the first arg, should keep the flexibility, means not just "mainnet" & "conquest", the path can be changed by user's input. e.g. if user input --network testnet, the CfgAion can set the path to "BASE_PATH + /config/testnet/xxx"

Reply:
what if the user's input was ./aion.sh --network testnet - there are no such path which contains the config.xml and genesis.json file; should i just default it back to mainnet